pom.xml

```diff
@@ -124,6 +124,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${apache.commons.lang3.version}</version>
```

Comment:
I recommend removing the version line and the variable **apache.commons.lang3.version**, because Spring Boot 2.6 already provides variable **commons-lang3.version** for this and also specifies in `<dependencyManagement>` to use that version, so specifying versions here just makes it harder ot maintain correct version.
```suggestion

```